docker-compose.yml

```
@@ -48,18 +49,45 @@ services:
     ports:
       - 7000 # cluster
       - 9042 # native
+    ports:
+      - 9042:9042
 
   test:
-    image: node:10.15.0
+    image: node:10.24.0
```

```suggestion
    image: node:12
```

might as well use a version of node that supports the new ssh key formats as well.